src/AtmosParameters.jl

```diff
@@ -13,16 +13,48 @@ AtmosSGS.c_2_KASM(ps::AbstractEarthParameterSet)        = AtmosSGS.c_e2_KASM(ps)
 AtmosSGS.c_3_KASM(ps::AbstractEarthParameterSet)        = AtmosSGS.c_a_KASM(ps)^(3/2)
 
 Microphysics = CLIMAParameters.Atmos.Microphysics
+
 # Microphysics parameters
-Microphysics.MP_n_0(::AbstractEarthParameterSet)          = 8e6 * 2
+Microphysics.n0_rai(::AbstractEarthParameterSet)          = 8e6 * 2
+Microphysics.n0_ice(::AbstractEarthParameterSet)          = 1e7 * 2
+Microphysics.μ_sno(::AbstractEarthParameterSet)           = 4.36 * 1e9
+Microphysics.ν_sno(::AbstractEarthParameterSet)           = 0.63
+
 Microphysics.C_drag(::AbstractEarthParameterSet)          = 0.55
+
+Microphysics.α_rai(::AbstractEarthParameterSet)           = 4/3. * π * CLIMAParameters.Planet.ρ_cloud_liq
```

Comment:
```suggestion
Microphysics.α_rai(ps::AbstractEarthParameterSet)           = 4/3. * π * CLIMAParameters.Planet.ρ_cloud_liq(ps)
```
This is a function, so we must call it with the parameter set